parm/atm/obs/config/sondes.yaml

```diff
@@ -1,74 +1,56 @@
 obs space:
   name: sondes
   obsdatain:
-    engine:
+    engine: 
       type: H5File
-      obsfile: $(OBS_DIR)/$(OBS_PREFIX)sondes.$(OBS_DATE).nc4
-    obsgrouping:
-      group variables: ["station_id"]
-      sort variable: "air_pressure"
-      sort order: "descending"
+      obsfile: sondes_obs_$(OBS_DATE).nc4
+      obsgrouping:
+        group variables: ["station_id"]
+        sort variable: "air_pressure"
+        sort order: "descending"
   obsdataout:
     engine:
       type: H5File
-      obsfile: $(DIAG_DIR)/diag_sondes_$(OBS_DATE).nc4
+      obsfile: sondes_diag_$(OBS_DATE).nc4
   io pool:
     max pool size: 1
-  simulated variables: [eastward_wind, northward_wind, air_temperature, specific_humidity, surface_pressure]
-get values:
```

Comment:
why are `get values` and `interpolation method` removed?

Reply:
@CoryMartin-NOAA fixed.